Sources/Commands/SwiftTool.swift

```diff
@@ -430,8 +424,7 @@ public class SwiftTool {
 
                 // Terminate all processes on receiving an interrupt signal.
                 DefaultPluginScriptRunner.cancelAllRunningPlugins()
```

Comment:
Looking forward to replacing this line with the new API you're adding (in a follow-on PR)! :)

Reply:
happy to try and do a follow up PR for that, need to learn more about how plugin cancellation could work